lib/node_modules/@stdlib/math/base/complex/signum/test/test.js

```diff
@@ -0,0 +1,150 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var PINF = require( '@stdlib/constants/math/float64-pinf' );
+var NINF = require( '@stdlib/constants/math/float64-ninf' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var isNegativeZero = require( '@stdlib/math/base/assert/is-negative-zero' );
+var isPositiveZero = require( '@stdlib/math/base/assert/is-positive-zero' );
+var Float64Array = require( '@stdlib/array/float64' );
+var csig = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof csig, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function calculates the signum function of argument', function test( t ) {
+	var expected;
+	var actual;
+
+	actual = csig( -4.2, 5.5 );
```

Comment:
For the expected values, I am not sure, if you did this already, but I checked Julia. The results match. To be clear that the expected values are not circular (i.e., the values that we test against are not values generated by the function being tested), I recommend adding a comment at L45 saying

```javascript
// Tested against Julia...
actual = csignum( -4.2, 5.5 );
...
```
Similarly at L69 and below.

Reply:
In general, when we add implementations, we test against reference implementations, such as NumPy/SciPy, Julia, R, Boost, etc. I am okay with a minimal set of tests, as done here, unless you want to provide more comprehensive testing out of a sense of thoroughness. If so, let me know, and I will provide a set of steps to get you started.

Reply:
Yes, I'll surely want to do that, not only for the sake of completeness but also for getting to know the testing method.

Reply:
Yes, it looks good. Thank you for the detailed explanation. I'll get back with the tests included in this PR itself?

Reply:
Yes, that is correct. You'll want to add the fixtures and tests to this PR.

Reply:
I've added the test against the fixtures. I have also kept the original test (with manually computed expected answers). Let me know if that is rendered redundant, I'll remove it if that's the case.

Reply:
Yeah, I would go ahead and remove the redundant test (i.e., L80-102).